### DISCUSSION TITLE
Demande particulière pour l'envoi du TP

### DISCUSSION THREAD

**Discussion by ghost** [dehboris/trombi-srs-2021#23] [General]
Bonjour Monsieur.J"ai fini tout mon travail .Mais je ne trouve pas l'onglet Create pull request pour envoyer le TP.Je fais comment?

**Comment by dehboris** [comment]
![pull request](https://user-images.githubusercontent.com/8873836/125652959-e37086cd-6f09-4f5b-a8f1-79acc2aeb3f6.PNG)
